addons/udes_stock/tests/plot_load_tests.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+from scipy.stats import spearmanr
+from collections import defaultdict
+from matplotlib import pyplot as plt
+import sys
+
+COEFFICIENT_THRESHOLD = 0.7
+PVAL_THRESHOLD = 0.001
+
+# Do we want to bootstrap this?
+def scaling_coefficient(x, y):
+    """Nuno's neat method"""
```

Comment:
Given that this is public code, this should probably have a more globally meaningful docstring